src/Form/BaseEventCommandType.php

```diff
@@ -15,6 +16,8 @@ class BaseEventCommandType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $needToPurifyHtml = $builder->getData() instanceof BaseEvent ? !$builder->getData()->isCitizenAction() : false;
```

Comment:
You should override the option in the child form instead of adding overhead on all of them.
In https://github.com/EnMarche/en-marche.fr/blob/master/src/Form/CitizenActionCommandType.php#L18, you can add:
```diff
 $builder
     ->remove('capacity')
+    ->add('description', TextareaType::class, [
+        'filter_emojis' => true,
+        'purify_html' => true,
+    ])
 ;
```

Reply:
Done